plugins/deployment_manager/ContractMap.ts

```diff
@@ -31,7 +31,9 @@ export async function getContractsFromAliases(
       address
     );
 
-    contracts.set(alias, contract);
+    if (contract) {
```

Comment:
This is so we don't accidentally wipe something if the contract is missing? When would it be missing?

Reply:
This was for the `USDC:admin` not being a contract on `mainnet`, but which we need to crawl for testnet (since we deploy/use the admin, though still not sure why we need that, but didn't change it). In general spider fully supports crawling/aliasing non-contract addresses too now, but this was the last spot that was breaking when we actually try to include some